Makefile

```diff
@@ -1,3 +1,17 @@
+ifeq ($(OS),Windows_NT)
+  SHELL := powershell.exe
+  .SHELLFLAGS := -NoProfile -Command
```

Comment:
```suggestion
  .SHELLFLAGS := -NoLogo -NoProfile -Command
```

Reply:
I don't get it. It doesn't seem to make any difference. The copyright text isn't shown anyway when using the makefile. I'll add it if you still want it.

Reply:
more of a "best practice" (avoiding unnecessary output could avoid accidental problems?)

Reply:
Sure, added.